File: src/main/java/gregtechlite/gtlitecore/client/renderer/UniversiumRenderBehavior.java
package gregtechlite.gtlitecore.client.renderer;

import net.minecraft.item.ItemStack;
import net.minecraftforge.fml.relauncher.Side;
import net.minecraftforge.fml.relauncher.SideOnly;

public interface UniversiumRenderBehavior extends ItemRendererManager
{

    float getCosmicOpacity();

    @SideOnly(Side.CLIENT)
    default float getCosmicOpacity(ItemStack stack, Object... args)
    {
        return getCosmicOpacity();
    }

}

File: src/main/kotlin/gregtechlite/gtlitecore/client/renderer/UniversiumRenderBehavior.kt
package gregtechlite.gtlitecore.client.renderer

import net.minecraft.item.ItemStack
import net.minecraftforge.fml.relauncher.Side
import net.minecraftforge.fml.relauncher.SideOnly

interface UniversiumRenderBehavior : ItemRendererManager
{
    @SideOnly(Side.CLIENT)
    fun getCosmicOpacity(): Float

    @SideOnly(Side.CLIENT)
    fun getCosmicOpacity(stack: ItemStack, vararg args: Any): Float = getCosmicOpacity()
}